segmentation/utils/spatial_analysis_utils.py

```diff
@@ -253,3 +253,48 @@ def calculate_enrichment_stats(close_num, close_num_rand):
     dims = ["stats", "rows", "cols"]
     stats_xr = xr.DataArray(stats_data, coords=coords, dims=dims)
     return stats_xr
+
+
+def compute_neighbor_counts(fov_data, dist_matrix, distlim, self_neighbor=True,
+                            cell_label_col="cellLabelInImage"):
+    """Calculates the number of neighbor phenotypes for each cell. The cell counts itself as a neighbor.
+
+    Args:
+        fov_data: data for the current fov, including the cell labels, cell phenotypes, and cell phenotype ID
+        dist_matrix: cells x cells matrix with the euclidian
+            distance between centers of corresponding cells
+        distlim: threshold for spatial enrichment distance proximity
+        self_neighbor: If true, cell counts itself as a neighbor in the analysis.
+        cell_label_col: Column name with the cell labels
+    Returns:
+        counts: data array with phenotype counts per cell
+        freqs: data array with phenotype frequencies of
```

Comment:
Change this to freqs.T, since we're returning the transpose of freqs.

Reply:
we are actually just returning freqs.  It looks like we're returning `freqs.T` but it's easier to broadcast the division for counts -> freqs within the transposed space and returned the transpose of that result (although maybe `num_neighbors` could be transposed instead?).  Regardless, saying that we're returning `freqs.T` makes it seem like the frequency output has to be transposed outside of the function.  I think it's best to leave it as is imo.

Reply:
Ah OK, thanks for clarifying.